docs/reference/technology/psa/psa.md

```diff
@@ -32,11 +32,14 @@ The Mbed implementation of PSA supports the following platform types:
 
```

Comment:
This page doesn't seem to cover architecture, so I think the content would be better here: https://os.mbed.com/docs/mbed-os/development/apis/security.html

Reply:
Updated as suggested.